src/Global.lua

```diff
@@ -285,23 +285,18 @@ function getOrderedPlayers()
         return {""}
     end
 
-    -- local players = {"White", "Yellow", "Teal", "Red"}
-    local players = seated_players
+    local clockwise_order = {"White", "Yellow", "Teal", "Red"}
     local ordered_players = {}
-    local i = math.random(player_count)
-    local count = 0
-    while count < player_count do
-        if (i > player_count) then
-            i = 1
-        end
+    local start_index = math.random(player_count)
 
-        local arcs_player = ArcsPlayer:new{
-            color = players[i]
-        }
-        -- ordered_players[color] = arcs_player
-        table.insert(ordered_players, arcs_player)
-        count = count + 1
-        i = i + 1
+    for i = 1, #clockwise_order do
+        local color = clockwise_order[(start_index + i - 2) % #clockwise_order + 1]
```

Comment:
If start_index is 1 this will result in a nil value. I don't think this will cause an error but will likely mess with the random distribution.

start_index=1, 1+1-2=0, 0%5=0, clockwise_order[0]=nil

Reply:
I made an error; mod takes precedence over addition. So, the math would be 1+1-2=0, 0%4=0, 0+1=1, clockwise_order[1]="White"